bin/http-server

```diff
@@ -48,11 +53,39 @@ var port = argv.p || parseInt(process.env.PORT, 10),
     ssl = !!argv.S || !!argv.ssl,
     proxy = argv.P || argv.proxy,
     utc = argv.U || argv.utc,
+    logfile = (function () {
+      var file = argv.logfile || argv.l;
+      if (!file) {
+        return '';
+      }
```

Comment:
Could we check for these options before jumping into this function, thus avoiding requiring evaluation at all?